steve/steve/AppModel.swift

```diff
@@ -78,15 +79,34 @@ class AppModel: ObservableObject {
           self.waitingForResponse = false
       }
     // TODO: once API rigged up should not just parrot response
-//    if let steveModel = steve {
-//      await makeSteveSay(text: text, steveModel)
-//    } else {
-//      print("❌ steve Entity global variable not set for getSteveResponse call, bailing")
-//    }
-//    
-//    DispatchQueue.main.async {
-//      self.waitingForResponse = false
-//    }
+
+    if let steveModel = steve {
+      if useLocalModel {
+        //      let promptText = "You are Steve Jobs, the founder of Apple Computer. \(text)"
+        let promptText = text
+        
+        // TODO: replace these with API-backed setup
+        if bot != nil {
+          print("Asking question to local LLM with prompt text: ", promptText)
+          let question = bot?.preProcess(promptText, [])
+          let answer = await bot!.getCompletion(from: question!)
+          print("Got answer from local LLM: ", answer)
+          await makeSteveSay(text: answer, steveModel)
+        } else {
+          print("Local LLM bot is nil, so just miming request for now")
+          await makeSteveSay(text: text, steveModel)
+        }
+      } else {
+        print("TODO: add server API request here")
```

Comment:
@kon-rad I will merge this, just need to do API request code here so we can safely toggle between server and local LLM.